gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -0,0 +1,650 @@
++
+Be careful to avoid selecting the similarly named policy for LDAP.
+. Click *Next*.
+. Configure a Basic Authentication policy:
++
+image::gateway-quickstart-config-basic-auth.png["Screenshot of 'Configure Basic Authentication - Simple policy' page",width=80%]
++
+.. For *User Name*, enter `user` instead of your real user name. 
++
+If the field is pre-populated with a value, you can keep the value or change it to the suggested value. 
++
+.. For *User Password*, enter `pw`.
++
+If the field is pre-populated with a value, you can keep the value or change it to the suggested value. 
++
+.. Click *Apply*.
+
+. Test the API instance:
+.. Use `kubectl` to get your external IP address again so that you can use it in the next step:
++
+[source,kubectl]
+----
+kubectl -n gateway get services
+----  
+.. Run the following `curl` a command _without_ authentication parameters, replacing `<your_external_ip>` with the `EXTERNAL-IP` returned by preceding step:
++
+[source,curl]
+----
+curl -s -o /dev/null -w "%{http_code}\n" --request GET 'http://<your_external_ip>:80/users'
+----
++
+The command returns a `401` because the URI lacks authentication parameters:
++
+----
+401
+----
++
+.. Run the following `curl` a command _with_ authentication parameters, replacing `<your_external_ip>` with your `EXTERNAL-IP` from the preceding step:
++
+[source,curl]
+----
+curl -s -o /dev/null -w "%{http_code}\n" --request GET 'http://<your_external_ip>:80/users' -u user:pw
+----
++
+The command prints the resulting `200` status code, indicating success:
++
+----
+200
+----
++
+The `user` and `password` parameters (`user:pw`) must match the values you specified when you applied the policy. If you enter incorrect authentication parameters, the following error occurs:
++
+----
+401
+----
++
+To address the error, run the command with the correct authentication parameters, or return to the policy configuration page for your API in API Gateway, edit the policy configuration, and rerun the `curl` commands.
++
+image::gateway-quickstart-k8-config-basic-auth-edit-dialogue.png["Screenshot of 'Configure Basic Authentication - Simple policy' edit dialogue",width=90%]
++
+The page for editing the credentials looks similar to this one:
++
+image::gateway-quickstart-k8-config-basic-auth-edit.png["Screenshot of 'Configure Basic Authentication - Simple policy' configuration page",width=90%]
+
+
+[[troubleshooting]]
+== Troubleshooting
+//iterative troubleshooting info to add as needed:
+
+If you encounter an error when running a command, check for troubleshooting information: 
+
+<<docker-troubleshooting>> | <<helm-troubleshooting>>
+
+[[docker-troubleshooting]]
+=== Docker Command Issues
+
+Troubleshoot errors returned by a `docker` command:
+
+[[daemon-bad-response]]
+* If Docker is not started, a `docker` command returns an error message similar to these:
++
+----
+Error response from daemon: Bad response from Docker engine
+----
++
+If you get this error, you need to start Docker.
```

Comment:
```suggestion
If you get this error, start Docker.
```

Reply:
implied is that the reason they are getting the error is that Docker isn't started, so they need to do that

Do we have guidance against "need to"? Probably do, huh.

Reply:
https://confluence.internal.salesforce.com/pages/viewpage.action?spaceKey=MTDT&title=MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22needto,%22%22haveto,%22%22must%22